android-java/app/src/main/java/com/example/msaldemo/MainActivity.java

```diff
@@ -0,0 +1,129 @@
+package com.example.msaldemo;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.ActionBarDrawerToggle;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.core.view.GravityCompat;
+
+import android.view.MenuItem;
+import android.view.View;
+
+import androidx.drawerlayout.widget.DrawerLayout;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+
+import com.google.android.material.navigation.NavigationView;
+
+public class MainActivity extends AppCompatActivity
+        implements NavigationView.OnNavigationItemSelectedListener,
+        OnFragmentInteractionListener{
+
+    enum AppFragment {
+        SingleAccount,
+        MultipleAccount,
+        B2C
```

Comment:
Are these fragments going to be supported in this same app? If not, I think we should remove them